### ISSUE TITLE
UI improvements

### ISSUE THREAD

**Issue by markvdb** [yoe/sreview#58]
The UI for setting start and end points could do with some improvements. Here's an example I stumbled upon recently that might give some ideas:

https://www.kapwing.com/editor/trimmer

**Comment by yoe** [comment]
Yeah, I'm aware of it not being ideal. Hence there's http://opensourcedesign.net/jobs/jobs/2018-02-11-ux-design-for-video-review-interface and @belenbarrospena 's work at https://belenbarrospena.github.io/sreview_design/

;-)

Won't close this until that gets implemented properly, but rest assured this is being worked on.

**Comment by belenbarrospena** [comment]
@markvdb thanks for the link! I will definitely check it out. 

I am posting work and progress at 
https://github.com/belenbarrospena/sreview_design/wiki

Feedback always welcomed.
